frontend/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+Jinja2 = "*"
+watchdog = "*"
+pyyaml = "*"
+argh = "*"
+
+[scripts]
+build = "python3 build.py ../config.json.example"
```

Comment:
I don't know if `pipenv` can do this, but could this take the path to the config file as an optional argument?  It would be nice for `pipenv run build` to use the sample one, and `pipenv run build ../config.json` to use the real one.

If that's not possible, how can I build it using the real config file?

Reply:
I added the build-prod script you can just pass the config afterwards

https://github.com/TypicalFence/lainonlife/blob/742246d4dffd6ae1182ab0dbf5e02ab41bd0513c/frontend/Pipfile#L17

should we rename build to build-dev and build-prod to just build?

Reply:
actually wait a bit with merging~ i think we can just add a bash script for that

Reply:
:+1:

Reply:
I'm setting up continuous deployment, so just let me know when you're done

Reply:
sorry it took so long, I got distracted I think this solution is the best one, the bash script calls build.py with `../config.json.example` if nothing is specified